docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -2,6 +2,19 @@
 Query management properties
 ===========================
 
+``query.execution-policy``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** ``phased``
+* **Session property:** ``execution_policy``
+
+The execution policy default value ``phased`` supports larger ETL workloads and
```

Comment:
Maybe `Phased execution schedule maximizes cluster resource utilization and provides lowest query wall time. Stages are scheduled in such a sequence that they shouldn't become blocked because of inter-stage dependency`?

`The
previous default value ``all-at-once``, schedules all the phases of a query at
one time.` -> `The
previous default value ``all-at-once``, schedules all the stage of a query at
one time.`

`The ``legacy-phased`` value schedules phases of a query only when it
can start processing them.`` -> `The ``legacy-phased`` schedules stages in such a sequence that they shouldn't become blocked because of inter-stage dependency. Compared to ``phased`` scheduler, `legacy-phased`` will try to minimize number of running stages and can increase query wall time`

Reply:
This was very helpful, I have a new version for review. Thanks @sopel39